ui-ngx/src/assets/help/en_US/widget/lib/map/clustering_color_fn.md

```diff
@@ -0,0 +1,64 @@
+#### Clustering marker function
+
+<div class="divider"></div>
+<br/>
+
+*function (data, childCount): string*
+
+A JavaScript function used to compute clustering marker color.
+
+**Parameters:**
+
+<ul>
+  <li><b>data:</b> Array<<code><a href="https://github.com/thingsboard/thingsboard/blob/5bb6403407aa4898084832d6698aa9ea6d484889/ui-ngx/src/app/modules/home/components/widget/lib/maps/map-models.ts#L108" target="_blank">FormattedData</a></code>> - A <a href="https://github.com/thingsboard/thingsboard/blob/5bb6403407aa4898084832d6698aa9ea6d484889/ui-ngx/src/app/modules/home/components/widget/lib/maps/map-models.ts#L108" target="_blank">FormattedData</a> object associated with marker or data point of the route.<br/>
```

Comment:
This description of the parameter is copy-pasted from some other place and is wrong. Please fix this before we can merge.